scenarioo-client/app/manage/label-colors/color-helpers.ts

```diff
@@ -0,0 +1,16 @@
+import * as contrast from 'contrast/index';
+
+export const hexadecimalColorIsValid = (color: string): boolean => {
+    // Color hexadecimal should have the format #000 or #0000000
+    const hexadecimalColorRegexPattern = /^#([0-9A-F]{3}){1,2}$/i;
+    return hexadecimalColorRegexPattern.test(color);
+};
+
+export const getContrastingForegroundColorOfBackgroundColor = (backgroundColor: string): string => {
```

Comment:
does not matter that it's a background color for this function. I would go with getContrastingColor(hexColor: string)

Reply:
Done